02-express-tutorial/Public/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<p>Hello Everyone.</p>
+<p>My name is Melanie.</p>
+
+<button id="fetchProductsButton">Fetch Products</button>
+    <div id="productDisplay"></div>
+
+
+<script>
```

Comment:
Nitpick (doesn't require change):
This script tag should have a [type="module"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type#module), which is a deferred JavaScript script.